service-csw/src/main/java/fi/nls/oskari/csw/helper/CSWISORecordDataQualityParser.java

```diff
@@ -10,16 +10,15 @@
 import javax.xml.xpath.*;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 
 public class CSWISORecordDataQualityParser {
 
     private static final Logger log = LogFactory.getLogger(CSWISORecordDataQualityParser.class);
     private static XPathExpression pathToLocalizedValue = null;
 
-    //Linage statement
-    private static XPathExpression XPATH_LINAGE_STATEMENT = null;
+    //Lineage statement
+    private static XPathExpression XPATH_LINEAGE_STATEMENT = null;
```

Comment:
Why are these fields static? They are modified within the constructor. Either make them final and init them in a `static { }` block outside of the constructor or make them non-static final and init them in the constructor.